src/pages/index.astro

```diff
@@ -1,33 +1,125 @@
+           title="Check out our subreddit">
+            subreddit
+        </a> or contribute to our community projects on
+        <a href={GITHUB_ORG_URL}
+           target="_blank"
+           title="Check out our subreddit">
+            GitHub
+        </a>!
+    </p>
+    <img src="/invite.png" alt="Search in the documentation">
+    <hr/>
 
-<script is:inline>
-	// Redirect your homepage to the first page of documentation.
-	// If you have a landing page, remove this script and add it here!
-	// window.location.pathname = `/en/introduction`;
-</script>
+    <h1>Contributing</h1>
+    <p>
+        We value and appreciate every contribution to our active projects!
+        If you see any issues, potential improvements in any of the
+        projects, listed below, or would like to add new content, feel
+        free to create
+        <a href="https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests">
+            pull requests
+        </a>!
+    </p>
+    <p>
+        If you'd like to help but unsure how, please check out the 'Issues'
+        tab in the GitHub repositories or ask
+        <a href={COMMUNITY_INVITE_URL}>on our Discord</a>.
+        We're always happy to help newcomers get started!
+    </p>
+
+    <h3>Active Projects</h3>
+    <ul>
+        <li>
+            <p>
+                Create new or update guides and documentations on this site using the
+                simple <a href="https://www.markdownguide.org/">Markdown</a> syntax!
+            </p>
+            <a href="https://github.com/syscraft-mc/syscraft.dev">https://github.com/syscraft-mc/syscraft.dev</a>
+        </li>
+        <li>
+            <p>Create new or update existing Discord bot commands for the <code>@Syscraft</code> bot!</p>
+            <a href="https://github.com/syscraft-mc/tags">https://github.com/syscraft-mc/tags</a>
+        </li>
+        <li>
+            <p>
+                Starter Server is a repository containing a basic Java Edition
+                Minecraft server to help people get started.
+                Help us improve it!
+            </p>
+            <a href="https://github.com/syscraft-mc/starter-server">https://github.com/syscraft-mc/starter-server</a>
+        </li>
+    </ul>
+</MainLayout>
 
 <style>
-	.logo {
-		display: flex;
-		align-items: center;
-		justify-content: center;
-		font-size: 4rem;
-		font-family: var(--font-mono);
-		text-transform: uppercase;
-		height: 10rem;
-	}
-
-	img {
-		height: 100%;
-		margin-right: 1rem;
-	}
+    .logo {
+        margin-top: 0;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        font-size: clamp(2rem, 10vw, 4rem);
+        font-family: var(--font-mono);
+        text-transform: uppercase;
+        height: 10rem;
+    }
+
+    .logo img {
+        height: 100%;
+        margin-right: 1rem;
+    }
+
+    @media only screen and (max-width: 500px) {
```

Comment:
It might be good to use a variable for the breakpoint here as it could become common usage for responsiveness around the site.

Reply:
So I looked into this, and for some reason, I could not get this to work by any means. Even just defining a variable on `:root` in either of the additional CSS files or in the `MainLayout` component, it showed that the variable has the correct value but the query would not work at all.

(I just tried it with some other property which worked fine)
![image](https://user-images.githubusercontent.com/46295973/236476022-03d1186a-bb32-4f39-91c9-68e3fc4c16e2.png)

I am not sure if I'm missing some Astro-specific behaviour or otherwise, but I'm a bit out of ideas with this.

Reply:
No worries, not a big deal :)